### PR TITLE
Connecting backend

### PR DESCRIPTION
**Pull request by ernurtorekul**

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -36,12 +39,37 @@ export default function Home() {
     setDays(generatedDays);
   }, []);
 
+  useEffect(() => {
+    const fetchSeries = async () => {
+      try {
```
Comment (zanz9) [inline app/page.tsx:47]: setLoading(true);

File: app/page.tsx [hunk 2]
```diff
@@ -36,12 +39,37 @@ export default function Home() {
...
+        const response = await fetch(
+          `${Endpoint.baseUrl}${Endpoint.seriesGetAll}`
+        );
+        if (!response.ok) {
+          throw new Error("failed fetching series");
+        }
+        const data = await response.json();
+        setSeries(data.data);
```
Comment (zanz9) [inline app/page.tsx:55]: and setSeriesCount

File: app/page.tsx [hunk 3]
```diff
@@ -36,12 +39,37 @@ export default function Home() {
...
+      } catch (err: any) {
+        setError(err.message);
+      } finally {
+        setLoading(false);
+      }
+    };
+    fetchSeries();
+  }, []);
+
   const totalPages = Math.ceil(series.length / itemsPerPage);
```
Comment (zanz9) [inline app/page.tsx]: use count

File: app/series/[id]/page.tsx [hunk 4]
```diff
@@ -16,39 +18,72 @@ import {
...
+  const [error, setError] = useState<string | null>(null);
 
   useEffect(() => {
-    if (id) {
-      const seriesDetail = series.find((s) => s.id === Number(id));
-      if (seriesDetail) {
-        setSelectedSeries(seriesDetail);
+    const fetchSeries = async () => {
+      if (!id) return;
+      const url = `${Endpoint.baseUrl}${Endpoint.seriesGet(Number(id))}`;
```
Comment (zanz9) [inline app/series/[id]/page.tsx:49]: use parse endoint

Comment (ernurtorekul) [reply app/series/[id]/page.tsx:49]: had problems using parse endpoint

**Pull request merged**